.github/workflows/minor-version-upgrade.yml

```diff
@@ -1,19 +1,18 @@
 name: Minor Version Upgrade Tests
-on:
-  push:
-    branches:
-      - BABEL_2_X_DEV
-  pull_request:
-    branches:
-      - BABEL_2_X_DEV
+on: [push, pull_request]
 
 jobs:
   extension-tests:
     env:
-      ENGINE_VER_FROM: BABEL_2_X_DEV__PG_14_2
-      EXTENSION_VER_FROM: BABEL_2_0_STABLE
+      ENGINE_VER_FROM: BABEL_2_X_DEV_DEFAULT_mVU_SOURCE
+      EXTENSION_VER_FROM: BABEL_2_X_DEV_DEFAULT_mVU_SOURCE
+      # TODO: hard-coding this means that we're not actually testing mVU for
+      # any PR that makes engine changes. This needs to be changed use
+      # something similar to .github/scripts/clone_engine_repo.
       ENGINE_VER_TO: BABEL_2_X_DEV__PG_14_2
```

Comment:
Target engine version should be `BABEL_2_1_STABLE__PG_14_3`.

Reply:
ugh, good catch. Really need to get this stuff to use the `clone_engine_repo` script. :(

Reply:
Fixed